com.unity.render-pipelines.core/CHANGELOG.md.preformat.bak

```diff
@@ -0,0 +1,227 @@
+# Changelog
```

Comment:
This file looks like it shouldn't be here :D

Reply:
FFFF